overops-functions/src/main/java/com/takipi/udf/microsoftteams/card/MicrosoftTextSection.java

```diff
@@ -0,0 +1,9 @@
+package com.takipi.udf.microsoftteams.card;
+
+public class MicrosoftTextSection implements MicrosoftSection {
+    private String text = "";
```

Comment:
This is only used when serializing as json, so IDEs mark it as unused with a warning.
Solution, make it `public final`
As a `String` object is immutable, there's no risk of it being internally modified even if final.

Also, as a rule of thumb, all members that _can_ be `final`, _should_ be `final`

Reply:
done